src/rdb.c

```diff
@@ -2901,12 +2902,12 @@ int rdbLoadRio(rio *rdb, int rdbflags, rdbSaveInfo *rsi) {
 
     if (empty_keys_skipped) {
         serverLog(LL_WARNING,
-            "Done loading RDB, keys loaded: %lld, keys expired: %lld, empty keys skipped: %lld.",
-                server.rdb_last_load_keys_loaded, server.rdb_last_load_keys_expired, empty_keys_skipped);
+            "Done loading RDB %s, keys loaded: %lld, keys expired: %lld, empty keys skipped: %lld.",
```

Comment:
update to: Done loading RDB file %s, keys loaded: %lld, keys expired: %lld, empty keys skipped: %lld.